app/src/app.js

```diff
@@ -3,39 +3,75 @@ const express = require('express');
 const connection = require('./model/connection');
 const app = express();
 const bodyParser = require('body-parser');
+const postValidation = require('./validation/post');
 app.use(bodyParser.json());
 
-app.get('/posts', async(req, res)=>{
-            let posts = await postModel.getPosts();
-            res.json(posts);
-        });
-app.get('/posts/:id', async (req, res) => {
```

Comment:
Z 5 minut się zastanawiałem czemu `get` zamienił się na `put`... Zauważyłem że poleciało to gdzieś niżej i jest całe na zielono, więc de facto nie wiem co się zmieniło. Prosiłem cię żeby te PRki zmieniały tylko to nad czym właśnie pracujesz, bo inaczej fatalnie się to analizuje

Reply:
przepraszam, chyba sobie z tym nie poradziłam.